assets/scss/frontend/elements/_reservation-details.scss

```diff
@@ -3,6 +3,7 @@ $reservation-border-radius: 6px;
 // Reservation
 .reservation-goto-checkout {
   margin-top: 1rem;
+  margin-bottom: 2rem;
```

Comment:
What's this for?

Reply:
Checkout button is sticky to list rooms on mobile. Above css fix that.